superset/connectors/sqla/models.py

```diff
@@ -704,7 +704,11 @@ def data(self) -> Dict[str, Any]:
             data_["fetch_values_predicate"] = self.fetch_values_predicate
             data_["template_params"] = self.template_params
             data_["is_sqllab_view"] = self.is_sqllab_view
-            data_["health_check_message"] = self.health_check_message
+            data_["health_check_message"] = (
```

Comment:
do not return health check message, when feature is disabled but had populated health check message before.

Reply:
Maybe add this comment to code as well?

```
# Don't return previously populated health check message in case
# the health check feature is turned off
```

Reply:
fixed.